en/Plugins/Publish.md

```diff
@@ -1,43 +1,49 @@
-The Publish plugin lets you publish notes from your vault onto your site, all within Obsidian.
+The Publish plugin lets you publish notes from your vault onto your site, all from within the Obsidian app.
 
-For more information about the Obsidian Publish service, what's included, how it works, and privacy go to the [[Obsidian Publish]] add-on service page.
+For more information about the Obsidian Publish™ service, what's included, how it works, and privacy, see the [[Obsidian Publish]] add-on service.
 
 ### Enable the plugin
 
-First, enable the Publish plugin. After that, you'll see the Publish icon appear in your [[Ribbon]], which looks like a paper plane.
+From within Obsidian > Settings > Plugin, enable the Publish plugin. After that, a Publish icon (which looks like a paper plane) will appear in the [[Ribbon]] pane.
 
 Once enabled, you can also set a hotkey for this option, or use the command palette to access it.
 
 ### Set up your site
 
-When purchasing Obsidian Publish, you'll be asked how many sites to purchase. You can then log in under Settings -> Account, so that the Publish plugin is aware of the number of remote sites you have.
+When purchasing Obsidian Publish, you'll be asked how many sites to purchase. Once you login under Settings > Account, the Publish plugin will display the number of publish sites you've purchased.
```

Comment:
Actually, this entire paragraph is no longer true for the current Publish plugin. You'll be able to create sites until you run out, but there's no "you have 3 sites" written anywhere. Let's just delete this paragraph?

Reply:
I'm totally in favor of removing the paragraph. But do we need to address the purchase options somewhere (ie. per user vs. per site)? Perhaps just on the purchase page itself.